.github/workflows/release-chart.yml

```diff
@@ -0,0 +1,59 @@
+name: Push Charts to helm-charts-registry
+
+on:
+  push:
+    tags:
+      - 'dbildungs-iam-*'
```

Comment:
please limit the tags a bit more. see example https://github.com/hpi-schul-cloud/infra-tools/blob/master/.github/workflows/bapontag.yaml